novawallet/Modules/Vote/Governance/TinderGov/Views/CardsZStack/VoteCardView.swift

```diff
@@ -46,7 +48,25 @@ final class VoteCardView: RoundedView {
         )
     }
 
-    private var viewModel: VoteCardViewModel?
+    private(set) var viewModel: VoteCardViewModel?
+
+    var isLoading: Bool = false {
+        didSet {
+            isSummaryLoading = isLoading
```

Comment:
That one looks strange: didStart/Stop loading changes both summary loading and isRequestedAmountLoading. As we want 2 parts to be independent then I would move them to the separate view and both views to the card. Also if we separate views we can also separate view model. It will reduce complexity that we talked about big single view model